src/Makefile

```diff
@@ -305,7 +305,7 @@ INFILES = \
 	../share/applications/linuxcnc-latency-histogram.desktop \
 	../scripts/linuxcnc_var \
 	../scripts/halcmd_twopass \
-	../scripts/runtests \
+	../scripts/runtests
```

Comment:
Just a ntoe -- this is a stylistic thing, and the backslash was intended. There are two alternatives and they're both bad when you want to split a list across many lines in a Makefile:
 * End every line with a `\` and always follow with a blank line
 * End every line but the last with a '\', no need to follow with a blank line

I wrote many but probably not all such blocks on our (Sub)Makefiles in the first style. I personally prefer it. If there's going to be such a stylistic change, I don't want to stand in the way of it, just giving background ..

Reply:
I likely did that without noticing it ;) Am not a fan of terminal '\'s since this gives the following empty line a(n unexpected to me) semantic.

I'll see that I remove that from the patch while rebasing.